app/components/header/style.scss

```diff
@@ -1,11 +1,39 @@
 .header {
   .menu {
     border-bottom: solid 1px #e6e6eb !important;
+    .item:first-child {
+      float: left;
+    }
+    .item:nth-child(2),
+    .item:nth-child(3) {
+      top: 50%;
+      float: right;
+    }
   }
 
   img {
     &:first-of-type {
       margin-right: 10px;
     }
   }
+  .ui.menu > .container {
+    display: block;
+  }
+}
+
+.button.ui.secondary.donate-button-header {
+  color: #fa451b;
```

Comment:
Should the styling of both register buttons be aligned? (On homepage? - we may be removing the index page so this may not be important)
![Screenshot 2021-05-26 at 10 13 28](https://user-images.githubusercontent.com/80352253/119634888-115a9980-be0b-11eb-8fb1-04383caa3dfe.png)